### PR TITLE
PR - Bonding Curves Outline

### PR DESCRIPTION
**Pull request by calchulus**

### CODE REVIEW

File: Outlines/Bonding Curves.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Bonding Curves
+- Bonding Curves are pricing curves that mediate the supply of a token. (include longer definition & elaboration)
```
Comment (ChrisChinchilla) [inline Outlines/Bonding Curves.md:2]: @calchulus Yes, and especially expand on why these might be useful to someone, what problem are they trying to solve?

File: Outlines/Bonding Curves.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Bonding curves play a crucial role in token economics for a given crypto ecosystem. (include longer elaboration)
+
+- Provide some examples of dApps, Cryptocollectibles, FOMO3D, and other things that feature bonding curves.
```
Comment (ChrisChinchilla) [inline Outlines/Bonding Curves.md:6]: Focus (as much as possible) on dapps that are in production, and have users. Which might be hard, but let's try :)

File: Outlines/Bonding Curves.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- Discussing the consequences of different growth rates, convexity, and functions
+     - exponential
+     - linear
+     - log
+- Making a comparison to Bitcoin’s fixed supply and how that is achieved.
+- Discuss incentive alignment for new participants and previous entrants
+- Potential integration with other governance mechanisms, market influence, and community levers of lnfluence
+- Exploring some sample solidity smart contracts that allow for some bonding curves to be fully automated.
```
Comment (ChrisChinchilla) [inline Outlines/Bonding Curves.md:15]: @calchulus And feel free to draw this examples from live projects if they make them available.

**Comment by ChrisChinchilla** [comment]
A couple of small comments to begin @calchulus , but otherwise good.

File: Outlines/Bonding Curves.md (hunk 1 above)
Comment (calchulus) [reply Outlines/Bonding Curves.md:2]: Yep sounds awesome to me

File: Outlines/Bonding Curves.md (hunk 2 above)
Comment (calchulus) [reply Outlines/Bonding Curves.md:6]: Nope, I think there are lots of good examples - happy to stick with oones already live

File: Outlines/Bonding Curves.md (hunk 3 above)
Comment (calchulus) [reply Outlines/Bonding Curves.md:15]: yes

**Comment by calchulus** [comment]
Thanks Chris for the comments, let me know when good to start

**Comment by ChrisChinchilla** [comment]
@calchulus Please get started, sorry didn't realise you were waiting for me!

**Comment by ChrisChinchilla** [comment]
@calchulus How are you going with this?

**Comment by calchulus** [comment]
Hey Chris, things are going well, I was just adding some new stuff with
this new Cheese Wizards game Dapper labs is putting up. Think it helps
illustrate with a more recent live example of some bonding curves :)

Hope to share a full draft in the next couple days

**Comment by calchulus** [comment]
added the new draft - let me know if this is in the wrong spot. Thanks!

**Comment by ChrisChinchilla** [comment]
@calchulus OK, reviewed. Good stuff! Made some small changes for grammar and clarity. If you're happy with things, then please post to Kauri with a good title, maybe some images etc.

**Comment by calchulus** [comment]
ok, appreciate the edits - I will read through one more time to make sure,
throw a couple additional images, and then get it up! Thanks again Chris

>

**Comment by calchulus** [comment]
published!

https://kauri.io/article/f7853f4914bd42b6bee19758a97b42ee/v1/an-introduction-to-bonding-curves

**Comment by ChrisChinchilla** [comment]
Great @calchulus I will get bounty paid out later this week, currently at conferences right niw.

**Comment by calchulus** [comment]
Amazing. Any other content you'd like me to get started on?

On Jun 4, 2019 5:19 PM, "Chris Chinchilla" <notifications@github.com> wrote:

Great @calchulus <https://github.com/calchulus> I will get bounty paid out
later this week, currently at conferences right niw.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/kauri-io/Content/pull/34?email_source=notifications&email_token=AD2K6RKHJELTRHFDX7V6K4TPYYXRDA5CNFSM4HCGUZOKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODW36U3Y#issuecomment-498592367>,
or mute the thread

**Pull request merged**